content/transcribing.fr.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Transcrire un contenu audio en texte"
+nav_title: "Transcrire un contenu audio en texte"
+
+lang: fr   # change "en" to lang code, here and 2 @@s below
+last_updated: 2020-01-02   # Change to date of translation YYYY-MM-DD (month in middle)
+translators:
+- name: "Sofia Ahmed"
+contributors: 
+- name: "Sandra Velarde Gonzalez (ETNIC)"
+
+permalink: /media/av/transcribing/fr   # Add lang to end /link/to/page/@@
+ref: /media/av/transcribing/   # Do not change this
+layout: default
+github:
+   repository: w3c/wai-media-guide
+   path: 'content/transcribing.fr.md'   # Add lang to the middle of the filename, e.g., index.@@.md
+
+resource:
+  ref: /media/av/
+navigation:
+  previous: /media/av/sign-languages/
+  next:     /media/av/acknowledgements/
+changelog: /media/av/changelog/
+acknowledgements: /media/av/acknowledgements/
+
+description: Fournit de l'aide pour transcrire un contenu audio en texte -- pour créer des légendes et des transcriptions pour l'accessibilité des contenus multimédia audios et vidéos.
+image: /content-images/wai-media-guide/social.png
+
+footer: >   # Translate words below, including "Date:" and "Editor:". (Do not update the date.)
+   <p><strong>Date :</strong> Mise à jour : 10 septembre 2019. JOURNAL DES MODIFICATIONS.</p>
+   <p><strong>Rédacteur :</strong> <a href="http://www.w3.org/People/Shawn">Shawn Lawton Henry</a>. REMERCIEMENTS : liste les contributeurs et les crédits.</p>
```

Comment:
We should not translate "ACKNOWLEDGEMENTS" as it is automatically translated in the process.